blocks/table-of-contents/block.json

```diff
@@ -0,0 +1,39 @@
+{
+	"apiVersion": 2,
+	"name": "cata/toc",
+	"version": "0.1.0",
+	"title": "Table Of Contents",
+	"category": "widgets",
+	"description": "Wdiget which adds a Table of Contents by checking for Heading 2 tags (h2) and creating a list of links to these sections wherever this block is placed.",
+	"keywords": [ "contents", "table of contents", "table of" ],
+	"supports": {
+		"html": false,
+		"reusable": false,
+		"customClassName": false
+	},
+	"example": {
+		"attributes": {
+			"content": "Table Of Contents"
+		}
+	},
+	"textdomain": "cata",
+	"script": "file:./build/script.js",
+	"editorScript": "file:./build/index.js",
+	"editorStyle": "file:./build/index.css",
+	"style": "file:./build/style-index.css",
+	"attributes": {
```

Comment:
@richdacuban Can you breakdown why we need these attributes? I don't see text align for font size adjustments in the editor. Do we need the content attribute even though the content is static in the editor?

Reply:
You're right, we don't need those for a widget block type with no internal block children. Removed.